spec/fortnox/api/types/housework_types_spec.rb

```diff
@@ -25,7 +28,7 @@
     let(:document) do
       Fortnox::API::Model::Order.new(
         customer_number: '1',
-        tax_reduction_type: tax_reduction_type,
+        tax_reduction_type: Object.const_get(tax_reduction_type),
```

Comment:
Hmmm, this looks weird ...

Reply:
Does it? I just pass the class as a string and use `Object.const_get` to "constantize" the string. We don't have the `constantize` method from Rails to use here...

Reply:
Before, I created variables for `TYPE_ROT` and `TYPE_RUT`. Now I stub them, which I need to do in an RSpec context, which means they are no longer declared when I run `it_behaves_like 'housework type', 'CONSTRUCTION', 'TYPE_ROT'` (`'TYPE_ROT'` is now a string, before it was a class).